src/mokka/src/test/java/pl/hycom/mokka/service/openapi/DefaultOpenApiParsingServiceTest.java

```diff
@@ -0,0 +1,98 @@
+package pl.hycom.mokka.service.openapi;
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import io.swagger.v3.oas.models.OpenAPI;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.Spy;
+import org.springframework.core.convert.ConversionService;
+import pl.hycom.mokka.service.openapi.pojo.StubMappingListWrapper;
+
+import java.util.Collections;
+import java.util.List;
+
+public class DefaultOpenApiParsingServiceTest {
+
+    private static final String EXAMPLE_JSON_SCHEMA = "{\n" +
+        "  \"openapi\": \"3.0.2\",\n" +
+        "  \"paths\": {\n" +
+        "    \"/pet\": {\n" +
+        "      \"put\": {\n" +
+        "        \"tags\": [\"pet\"],\n" +
+        "        \"summary\": \"Update an existing pet\",\n" +
+        "        \"description\": \"Update an existing pet by Id\",\n" +
+        "        \"operationId\": \"updatePet\",\n" +
+        "        \"responses\": {\n" +
+        "          \"200\": {\n" +
+        "            \"description\": \"Successful operation\",\n" +
+        "            \"content\": {\n" +
+        "              \"application/xml\": {\n" +
+        "                \"example\": {\n" +
+        "                  \"versions\": [{\n" +
+        "                      \"status\": \"CURRENT\",\n" +
+        "                      \"updated\": \"2011-01-21T11:33:21Z\",\n" +
+        "                      \"id\": \"v2.0\",\n" +
+        "                      \"links\": [{\n" +
+        "                        \"href\": \"http://127.0.0.1:8774/v2/\",\n" +
+        "                        \"rel\": \"self\"\n" +
+        "                      }]\n" +
+        "                    },\n" +
+        "                    {\n" +
+        "                      \"status\": \"EXPERIMENTAL\",\n" +
+        "                      \"updated\": \"2013-07-23T11:33:21Z\",\n" +
+        "                      \"id\": \"v3.0\",\n" +
+        "                      \"links\": [{\n" +
+        "                        \"href\": \"http://127.0.0.1:8774/v3/\",\n" +
+        "                        \"rel\": \"self\"\n" +
+        "                      }]\n" +
+        "                    }\n" +
+        "                  ]\n" +
+        "                }\n" +
+        "              }\n" +
+        "            }\n" +
+        "          }\n" +
+        "        }\n" +
+        "      }\n" +
+        "    }\n" +
+        "  }\n" +
+        "}";
+
+    @Spy
+    private ConversionService conversionService;
+
+    @InjectMocks
+    private DefaultOpenApiParsingService testedService = new DefaultOpenApiParsingService();
+
+    @BeforeEach
+    public void setUp() {
+
+        MockitoAnnotations.initMocks(this);
```

Comment:
not needed when `@InjectMocks` is used. On class level you should add: `@ExtendWith(MockitoExtension.class)`